dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java

```diff
@@ -75,4 +77,15 @@ public static <T> PageInfo<T> of(IPage<T> iPage) {
     public static <T> PageInfo<T> of(Integer currentPage, Integer pageSize) {
         return new PageInfo<>(currentPage, pageSize);
     }
+
+    public Integer getTotalPage() {
```

Comment:
This seems to achieve the same function, and is simpler
this.totalPage = Math.ceil(this.total / this.pageSize);

Reply:
@DaZuiZui 
The original code is from a previously deleted section [#15181](https://github.com/apache/dolphinscheduler/pull/15181/files#diff-c367978011e1fa572823ec7cec246f9a21ad88699ef9c764ee39e9c96d23914b). But your suggestions are better, there is no reason not to optimize, you are welcome to create a PR❤️.

Reply:
> @DaZuiZui The original code is from a previously deleted section [#15181](https://github.com/apache/dolphinscheduler/pull/15181/files#diff-c367978011e1fa572823ec7cec246f9a21ad88699ef9c764ee39e9c96d23914b). But your suggestions are better, there is no reason not to optimize, you are welcome to create a PR❤️.

+1

Reply:
I have submitted PR
https://github.com/apache/dolphinscheduler/pull/15620